examples/item-without-collection.json

```diff
@@ -0,0 +1,163 @@
+{
+  "stac_version": "1.0.0-beta.2",
```

Comment:
Wasn't there an item without collection already above? Just wondering whether a second example for it is needed?

Reply:
Oh, shoot - I thought I got that one out of version control. Removing it.